tslint.json

```diff
@@ -1,6 +1,10 @@
 {
   "extends": "tslint-config-airbnb",
   "rules": {
-    "variable-name": false
+    "variable-name": false,
+    "no-this-assignment": false,
+    "import-name": false,
+    "max-line-length": false,
+    "no-else-after-return": false
```

Comment:
These rules were turned off in order to keep your code intact. They may be turned on if you desire.